wsmaster/codenvy-hosted-api-system-license/src/test/java/com/codenvy/api/license/server/jpa/SystemLicenseTckModule.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *  [2012] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+package com.codenvy.api.license.server.jpa;
+
+import com.codenvy.api.license.server.dao.SystemLicenseActionDao;
+import com.codenvy.api.license.server.model.impl.SystemLicenseActionImpl;
+import com.google.inject.TypeLiteral;
+import com.google.inject.persist.jpa.JpaPersistModule;
+
+import org.eclipse.che.commons.test.tck.JpaCleaner;
+import org.eclipse.che.commons.test.tck.TckModule;
+import org.eclipse.che.commons.test.tck.TckResourcesCleaner;
+import org.eclipse.che.commons.test.tck.repository.JpaTckRepository;
+import org.eclipse.che.commons.test.tck.repository.TckRepository;
+import org.eclipse.che.core.db.DBInitializer;
+import org.eclipse.che.core.db.schema.SchemaInitializer;
+import org.eclipse.che.core.db.schema.impl.flyway.FlywaySchemaInitializer;
+
+import static org.eclipse.che.commons.test.db.H2TestHelper.inMemoryDefault;
+
+/**
+ * @author Anatolii Bazko
+ */
+public class SystemLicenseTckModule extends TckModule {
+
+    @Override
+    protected void configure() {
+        install(new JpaPersistModule("main"));
+        bind(SchemaInitializer.class).toInstance(new FlywaySchemaInitializer(inMemoryDefault(), "che-schema", "codenvy-schema"));
+        bind(DBInitializer.class).asEagerSingleton();
+        bind(TckResourcesCleaner.class).to(JpaCleaner.class);
```

Comment:
Please replace this line with `bind(TckResourcesCleaner.class).to(H2JpaCleaner.class);` [source](https://github.com/eclipse/che/blob/92ba698b9a3879c2e346097ee7599d7eeb770dfa/wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/jpa/WorkspaceTckModule.java#L45-L45)